### PR TITLE
Publish Stash 0.6.2 chart

### PR DESCRIPTION
**Pull request by tamalsaha**
https://github.com/appscode/stash/releases/tag/0.6.2

### CODE REVIEW

**Comment by whereisaaron** [comment]
@tamalsaha could you consider adding my small PR #3234 to that updated chart kubernetes/charts#125?

**Comment by tamalsaha** [comment]
@whereisaaron, sorry I don't follow. Your fix for #3234 is included here:

https://github.com/kubernetes/charts/pull/3235/files#diff-229224d06e89c6cc6c5502d075e1de71R15

**Comment by whereisaaron** [comment]
@tamalsaha yes indeed! Sorry, I saw that later and thus deleted my comment.

**Comment by tamalsaha** [comment]
np :)

File: stable/stash/templates/cluster-role.yaml [hunk 1]
```diff
@@ -26,21 +26,32 @@ rules:
   resources: ["*"]
   verbs: ["*"]
 - apiGroups:
-  - extensions
+  - apps
```
Comment (mattfarina) [inline stable/stash/templates/cluster-role.yaml:29]: Switching to apps now means the chart will only work with a 1.9 cluster. we'd like the charts to work for N and N-1 releases (1.9 and 1.8) at the least. Beta APIs are supported for 2 releases or 6 months at the moment.

Comment (whereisaaron) [reply stable/stash/templates/cluster-role.yaml:29]: Hi @mattfarina. Thanks for reviewing the PR. I'm using the chart with 1.8 and RBAC successfully. I believe 1.8 does have the `apps` API group for deployments? It is `apps/v1beta2` in 1.8 and `apps/v1` in 1.9, but both versions have it. The `extensions` API group is actually deprecated in 1.8:
https://kubernetes.io/docs/reference/workloads-18-19/

Comment (tamalsaha) [reply stable/stash/templates/cluster-role.yaml:29]: @mattfarina , `apps/v1beta1` was added in 1.6 https://v1-6.docs.kubernetes.io/docs/concepts/workloads/controllers/deployment/

Comment (mattfarina) [reply stable/stash/templates/cluster-role.yaml:29]: Ah, yeah... somehow I was thinking it put it to app/v1. You're right.

**Pull request merged**